### ISSUE TITLE
title and name instead of label?

### ISSUE THREAD

**Issue by eroux** [buda-base/public-digital-library#224]
There is a suggestion that we display "name" instead of "label" (perfLabel / altLabel) for persons, and "title" instead of "label" for bibliographic entities... this is quite reasonable but is also less precise. Let's not do it yet but it's worth tracking

**Comment by xristy** [comment]
How is it less precise? It is more specific to me. I say "my name is ..." and ask for "the book with title ...". It would not use label in either situation.

**Comment by eroux** [comment]
Sure, I agree

**Comment by JannTibetan** [comment]
I agree with Chris and would like to see it changed. It's an easy way to increase the appearance of polish and user friendliness

**Comment by eroux** [comment]
in that case bibliographical records are:
- instances (bdr:MW...)
- works, image instances (bdr:W...)
- etext instances (bdr:IE...)
- etexts (bdr:UT...)

**Comment by berger-n** [comment]
done:
- http://library-dev.bdrc.io/show/bdr:P1583

![image](https://user-images.githubusercontent.com/33383857/169835431-6385c2b7-1372-4ba5-ac11-2c2f229969a3.png)

---

- http://library-dev.bdrc.io/show/bdr:WA0BC001

![image](https://user-images.githubusercontent.com/33383857/169835566-dff915e5-339a-413e-b6d0-58f32dfb0cd1.png)

---

- http://library-dev.bdrc.io/show/bdr:MW3CN1317

![image](https://user-images.githubusercontent.com/33383857/169835729-61488ef5-7ec3-481f-9e92-0d21be61da3c.png)

**Comment by berger-n** [comment]
fixed case of other non-bibliographic entities like http://library-dev.bdrc.io/show/bdr:G844

![image](https://user-images.githubusercontent.com/33383857/169843525-e6da0f70-9d3b-468e-8f3b-30369a39a6f6.png)
